429.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+from collections import deque
+
+"""
+# Definition for a Node.
+class Node:
+    def __init__(self, val=None, children=None):
+        self.val = val
+        self.children = children
+"""
+
+class Solution:
+    def levelOrder(self, root: 'Node') -> List[List[int]]:
+        levels = []
+        
+        if not root:
+            return levels
+        
+        queue = deque([root])
+        prev, curr, level = 1, 0, []
+        
+        while queue:
```

Comment:
Try to re-write this using the template I described [here](https://github.com/woke7/leetcode-python/pull/19#discussion_r606710471).